src/components/StarRating/src/StarRating.vue

```diff
@@ -0,0 +1,157 @@
+<template>
+	<div :style="starComputedStyles">
+		<star
+			v-for="star in STAR_COUNT"
+			:key="star"
+			:fill="fillForStarRating(star)"
+			:color="color"
+			:class="$s.Star"
+			@mouseenter="hoverStar(star)"
+			@mouseleave="unhoverStar(star)"
+			@click="clickStar(star)"
+		/>
+	</div>
+</template>
+
+<script>
+import chroma from 'chroma-js';
+import Star from './Star.vue';
+
+const MIN_RATING = 0;
+const MAX_RATING = 5;
+const HALF_RATING = 0.5;
+// eslint-disable-next-line no-magic-numbers
+const STAR_COUNT = 5;
```

Comment:
Wouldn't the STAR_COUNT always be the same as the MAX_RATING?

Reply:
Yeah it could be haha, I'll merge them.